packages/store-api/codegen.yml

```diff
@@ -0,0 +1,9 @@
+overwrite: true
+schema: "./src/typeDefs/*.graphql"
+documents: null
+generates:
+  src/__generated__/schema.ts:
```

Comment:
Should we add this to `.gitignore`? 🤔

Reply:
not really, we export these typings, so it's good to have this on our repo